grpc-java/grpc-mtls/build.gradle

```diff
@@ -7,4 +7,5 @@ description = 'Utility project to make x509 client cert available in grpc calls
 dependencies {
     Classpaths.inheritGrpcPlatform(project)
     implementation 'io.grpc:grpc-core'
+    implementation 'io.grpc:grpc-services'
```

Comment:
None of the grpc-services dependencies added seem necessary. I've removed all of them and things still compile and run. server has an api dependency on them w/ the comment:

```
    api('io.grpc:grpc-services') {
        because 'downstream dagger compile, see deephaven-core#1722'
    }
```

What's the reason you've added them as deps?

Reply:
Huh, IDE had been complaining about not being able to find HealthStatusManager, but if you removed them and it's happy, I'll remove them too.

Reply:
Removed them, and it still works.